main.go

```diff
@@ -74,6 +76,10 @@ func main() {
 		addrs, passwords, aliases = loadRedisArgs(*redisAddr, *redisPassword, *redisAlias, *separator)
 	}
 
+	if *useCfBinding {
```

Comment:
nice, this is a lot better.

Can you make this part of the if block 67-77 and turn that into a switch statement?

Reply:
How would I put this into a switch statement?

Reply:
```
switch {
    case *redisFile != "": ....

    case *useCfBinding: ....

    default: ....


}
```

Reply:
Done